docs/test/deploy-from-settings.md

```diff
@@ -0,0 +1,46 @@
+Open deploy dialog (+) and select "Specify app details below"
+
+## Expected result at execution
+* In positive case the "Deploy" button must lead to the creation of a running application
+* Validation errors affecting particular fields must be displayed beneath these fields. In case the deployment  fails due to an exception, then a pop dialog is shown
+
+## Specific factors
+
+
+|Factor                    |1                                     |2                                  |3                                                        |4                                  |5                                            |Comment                                                                         |
+|--------------------------|--------------------------------------|-----------------------------------|---------------------------------------------------------|-----------------------------------|---------------------------------------------|--------------------------------------------------------------------------------|
+|App name                  |Max-length                            |Same name RC exists                |Same name RC does not exists and same name Service exists|Same name exists in other namespace|                                             |App name is used in the help text which does have some impact on layouting.     |
+|Container image           |Max-length                            |With version                       |With hostname of registry                                |With port of registry              |                                             |                                                                                |
+|Number of pods            |Floating point number                 |Not a number                       |Max-values                                               |                                   |                                             |                                                                                |
+|Port                      |Border cases(1, 65535)                |Out of range(<0, >65535)           |Floating point number                                    |Not a number                       |Same port is mapped to different target ports|                                                                                |
+|Protocol                  |TCP                                   |UDP                                |                                                         |                                   |                                             |                                                                                |
+|Exporse service externally|lcoal                                 |vagrant or OpenStack               |GCE                                                      |                                   |                                             |                                                                                |
```

Comment:
Sorry, I forgot...better this way

local | without load-balancer (vagrant) | with load-balancer (GCE)